### PR TITLE
Test: Add Follow & Collect NFT Implementation Initialization Tests

### PR DESCRIPTION
**Pull request by Zer0dot**
This PR adds just a couple new tests to ensure that follow & collect NFT implementations cannot be initialized. In particular, these tests may help if/when the implementations are updated in this repository to ensure this invariant holds.

### CODE REVIEW

**Comment by donosonaumczuk** [comment]
I think it looks good. Before approving/merging we should think if we want to add `master` and `development` branches, to differentiate current deployed code from under-development one. In that case, this PR should go into a new `development` branch.

**Review by mnlc — APPROVED** [review]

**Pull request closed without merging**
